daprdocs/content/en/developing-applications/local-development/multi-app-dapr-run/multi-app-overview.md

```diff
@@ -0,0 +1,85 @@
+---
+type: docs
+title: Run multiple applications with one command
+linkTitle: Multi-app Run overview
+weight: 1000
+description: Learn the scenarios around running multiple applications with one Dapr command
+---
+
+{{% alert title="Note" color="primary" %}}
+ Multi-app `dapr run -f` is currently a preview feature only supported in Linux/MacOS. 
+{{% /alert %}}
+
+Let's say you want to run several applications locally to test them together, similar to a production scenario. With a local Kubernetes cluster, you'd be able to do this with helm/deployment YAML files. You'd also have to build them as containers and set up Kubernetes, which can add some complexity. 
+
+Instead, you simply want to run them as local executables in self-hosted mode.  However, self-hosted mode requires you to:
+
+- Run multiple `dapr run` commands
+- Keep track of all ports opened (you cannot have duplicate ports for different applications)- Remember the resources folders and configuration files that each application refers to
+- Recall all of the additional flags you used to tweak the `dapr run` command behavior (`--app-health-check-path`, `--dapr-grpc-port`, `--unix-domain-socket`, etc.)
+
+With Multi-app Run, you can easily start multiple applications in self-hosted mode using a single `dapr run -f` command.
+
+## Multi-app Run template file
+
+When you execute `dapr run -f`, Dapr parses the multi-app template file initialized with `dapr init`. By default, this template file is called `dapr.yaml`. 
+
+You can customize the template file by executing `dapr run -f ./<your-preferred-file-name>.yaml`.
+
+The following `dapr.yaml` example includes some of the configurations you can customize to your applications. For a more in-depth example and explanation of variables, see [Multi-app template]({{< ref multi-app-template.md >}}).
```

Comment:
I suggest that we use the term "template properties" rather than template configuration, which can cause more configuration confusion. 

```suggestion
The following `dapr.yaml` example includes some of the template properties you can customize for your applications. For a more in-depth example and explanation of the template properties, see [Multi-app template]({{< ref multi-app-template.md >}}).
```